src/components/avatars/OcAvatarFederated.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <oc-avatar-item
+    icon="filter_drama"
+    icon-color="#5AAB9F"
+    :name="name"
+    :accessible-label="accessibleLabel"
+  ></oc-avatar-item>
+</template>
+
+<script>
+export default {
+  name: "OcAvatarFederated",
+  status: "ready",
+  release: "2.1.0",
+
+  props: {
+    /**
+     * Name of the public link used as an accessible label
```

Comment:
It's not necessarily a public link if it's a federated share if I'm not mistaken?